pylintrc

```diff
@@ -1,3 +1,4 @@
 [BASIC]
 # Allow django's urlpatterns, and our log preference
 const-rgx = (([A-Z_][A-Z0-9_]*)|(__.*__)|log|urlpatterns)$
+ignore=migrations
```

Comment:
Okay, so adding `#pylint: skip-file` to the migration file fixes it. So I'll add that to all migrations for now.